main.py

```diff
@@ -8,7 +8,30 @@
 from steps import STEPS
 from db import DB, DBs
 import typer
+from dotenv import load_dotenv
 
+load_dotenv()
+
+api_key = os.getenv("OPENAI_API_KEY") 
```

Comment:
We can use `os.environ[key]` rather than `os.getenv(key)` to look up mandatory keys.

`environ[key]` raises a KeyError if key does not exist in the environment.  `KeyError(key)`

The current code does allow setting a custom exception message.

Reply:
I don't see much difference @spazm 

I mean @yhyu13 is checking if the kye exists or not below, in line 17.

Although I agree something like this would be shorter and easier to read/follow thoughts?

```python
try:
    openai.api_key = api_key
except KeyError:
    raise ValueError("OPENAI_API_KEY is not defined")
    
```

@yhyu13 can you add something like this?